class OuputNotBeZeroException extends Throwable
{
	String s;
	OuputNotBeZeroException(String str)
	{
		System.out.println("in OuputNotBeZeroException class");
		s=str;
	}
	public void showErr()
	{
		System.out.println(s);
	}
	
}
public class Assign_15
{
	public int ReturnData(int i,int j)
	{
		int div=0;
		try
		{
			div =i/j;
			System.out.println("In Try block");
		}
		catch(ArithmeticException e)
		{
			System.out.println(e.getMessage());
		}
		return div;
	}
	public static void main(String [] args) 
	{
		Assign_15 ss = new Assign_15();
		System.out.println(ss.ReturnData(4, 0));
		
	}

}
